### ISSUE TITLE
Minechem and Mekanism gases and liquids are incompatible

### ISSUE THREAD

**Issue by Moorleiche2k** [jakimfett/Minechem#695]
As title says. I also informed AdianCBrady about the issue and got the response that it seems up to you. Here the link: https://github.com/aidancbrady/Mekanism/issues/2169

**Comment by hilburn** [comment]
Yes they are incompatible - partially because Minechem chemicals are much easier to obtain and it would massively unbalance Mekanism.

Secondly, in order to interface with the Rotary Condensentrator - we can't, we would have to replace our own chemicals with Mekanism's, which would break a lot of our stuff, or replace the fluid registered for the Mekanism gas with our own, which presumably would break a lot of mekanisms stuff (eg. any existing world with mekanism hydrogen fluid would be unable to use it)

If @aidancbrady added the ability to register additional Fluid->Gas pairings to the Rotary Condensentrator to the API then we could do that, but nothing we can do until then

**Comment by Moorleiche2k** [comment]
Can you suggest this to him? I am not experienced with programming and especially not with modding minecraft. It would be wonderful to have this option, best in form of a config file, so that server owners may decide about unbalanced stuff.
Thanks

**Comment by jakimfett** [comment]
This is something we can possibly look at resolving for v6, but no promises about it...it may or may not ever happen.

**Comment by Moorleiche2k** [comment]
Awesome. I know of the difficulty of intercompatibility between mods myself (did some some fun stuff myself with modding Skyrim). So, many thanks for considering this and crossed fingers that it might become realized ;)

**Comment by jakimfett** [comment]
Yup, it can be a bit tricky. Thanks for your feedback, here's hoping it'll work!

(I'm going to leave this open as a v6 enhancement request.)
